sms.info.yml

```diff
@@ -8,3 +8,5 @@ configure: admin/config/smsframework
 dependencies:
   - telephone
   - dynamic_entity_reference
+test_dependencies:
```

Comment:
The only way to verify is to commit without the test dependencies and see if the tests pass. My previous experience with testbots required this line. Maybe DrupalCI doesn't need them, not sure.